setup/install/index.md

```diff
@@ -6,12 +6,9 @@ sidebar:
 redirect_from: /docs/installation-tools
 ---
 
-This section has the steps for installing and configuring Spinnaker.
-
-Using this procedure, the Spinnaker you install is suitable for use in production. 
-It's more configurable and available than a [Quickstart](/setup/quickstart/)
-installation (though that's a good choice if you just want to evaluate Spinnaker 
-without much work). 
+This section has the steps for installing and configuring Spinnaker on a path toward use in
+production. If you just want to evaluate Spinnaker without much work, one of the options
+in [Quickstart](/setup/quickstart/) might be a better choice.
```

Comment:
This section describes how to install and set up Spinnaker so that it can be configured for use in production.

Reply:
Your phrasing implies that at the end of this section, you are distinctly not yet set up for production, which is not always true.

Reply:
I am kind-of leaning toward's @dorbin's phrasing, since basic things you'd expect in production (a public endpoint, monitoring) aren't covered in this exact section

Reply:
Your call, Steven.

Reply:
@dorbin done, ptal